docs/framework/data/adonet/asynchronous-programming.md

```diff
@@ -36,17 +33,15 @@ This topic discusses support for asynchronous programming in the [!INCLUDE[dnprd
   
  For more information about the asynchronous programming feature that was introduced in [!INCLUDE[net_v45](../../../../includes/net-v45-md.md)], see:  
   
--   [Visual Studio Asynchronous Programming](http://go.microsoft.com/fwlink/?LinkId=220765)  
-  
--   [Using SqlDataReader’s new async methods in .Net 4.5 (Part 1)](http://blogs.msdn.com/b/adonet/archive/2012/04/20/using-sqldatareader-s-new-async-methods-in-net-4-5-beta.aspx)  
-  
--   [Using SqlDataReader’s new async methods in .Net 4.5 (Part 2)](http://blogs.msdn.com/b/adonet/archive/2012/07/15/using-sqldatareader-s-new-async-methods-in-net-4-5-beta-part-2-examples.aspx)  
-  
+- [Asynchronous programming in C#](../../../csharp/async.md)
+
+- [Asynchronous Programming with Async and Await (Visual Basic)](../../../visual-basic/programming-guide/concepts/async/index.md)
+ 
  When your user interface is unresponsive or your server does not scale, it is likely that you need your code to be more asynchronous.  Writing asynchronous code has traditionally involved installing a callback (also called continuation) to express the logic that occurs after the asynchronous operation finishes. This complicates the structure of asynchronous code as compared with synchronous code.  
   
  You can now call into asynchronous methods without using callbacks, and without splitting your code across multiple methods or lambda expressions.  
   
- The `async` modifier specifies that a method is asynchronous. When calling an `async` method, a task is returned.  When calling an `await` statement against the task, the current method exits immediately. When the task finishes, execution resumes in the same method.  
+ The `async` modifier specifies that a method is asynchronous. When calling an `async` method, a task is returned.  When calling an `await` statement against the task, the current method exits immediately. When the task finishes, execution resumes in the same method.
```

Comment:
nit: "calling an `await` statement makes no sense. Perhaps "When the `await` operator is applied to a task,"?

Reply:
I didn't read the rest of the text while I was fixing the links